src/controller/java/CHIPDeviceController-JNI.cpp

```diff
@@ -829,7 +866,7 @@ CHIP_ERROR IsWildcardChipPathId(jobject chipPathId, bool & isWildcard)
     VerifyOrReturnError(idType != nullptr, CHIP_JNI_ERROR_NULL_OBJECT);
     JniUtfString typeNameJniString(env, typeNameString);
 
-    isWildcard = strncmp(typeNameJniString.c_str(), "WILDCARD", 8);
+    isWildcard = strncmp(typeNameJniString.c_str(), "WILDCARD", 8) == 0;
```

Comment:
Guessing this was a bug fix?
Call it out in the description or maybe separate as a separate fix PR.

Reply:
Added to description.